Assets/Scripts/Util/Ladder.cs

```diff
@@ -23,11 +23,12 @@ private void AdaptTexture()
         var sizeDelta = ladderBottom.transform.localScale.y + ladderTop.transform.localScale.y;
```

Comment:
Noticed I made a little error here in factoring in the ladder bottom, if we want the ladder texture + hitbox to start from the bottom (and include it), we can just add the localScale.y from the ladderTop.transform
```suggestion
        var sizeDelta = ladderTop.transform.localScale.y;
```
This probably isn't perfect since I whipped this script up quickly in the first place but it looks nice for the application as of now